package.json

```diff
@@ -124,7 +124,7 @@
     "ovh-module-exchange": "^9.3.4",
     "ovh-module-office": "^7.0.7",
     "ovh-module-sharepoint": "^7.1.4",
-    "ovh-ui-angular": "^2.24.1",
+    "ovh-ui-angular": "^2.25.1",
```

Comment:
To avoid bad surprise, can you bump also `ovh-ui-kit ` to `v2.25.1`?

Reply:
Have bumped both to the latest releases which is 2.26.1